java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -95,13 +95,7 @@ public Table(long[] cudfColumns) {
       nativeHandle = createCudfTableView(views);
       this.rows = columns[0].getRowCount();
     } catch (Throwable t) {
-      for (int i = 0; i < cudfColumns.length; i++) {
-        if (this.columns[i] != null) {
-          this.columns[i].close();
-        } else {
-          ColumnVector.deleteCudfColumn(cudfColumns[i]);
-        }
-      }
+      ColumnView.cleanupColumnViews(cudfColumns, this.columns);
```

Comment:
need to zero-out `cudfColumns[i]` as we iterate in the loop that creates the column vectors. I would put this try catch around the creation of the column vectors for `this.columns`.

The views later, that's a different set of failures I believe.

Reply:
I don't want to change too much of this function, and the only reason I touched it was to refactor the cleanup code into a separate method.

Reply:
I will push an update to the zeroing out of the cudfColumns pointers